llvm/test/Transforms/LoopDataPrefetch/RISCV/basic.ll

```diff
@@ -0,0 +1,44 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 3
+; RUN: opt -mtriple=riscv64 -riscv-enable-loop-data-prefetch \
```

Comment:
Do you need `-riscv-enable-loop-data-prefetch`?

Reply:
No need. I will delete it.